libs/application/templates/financial-aid/src/fields/HomeCircumstancesForm/HomeCircumstancesForm.tsx

```diff
@@ -0,0 +1,128 @@
+import React, { useState } from 'react'
+import { Box, Input } from '@island.is/island-ui/core'
+import { homeCircumstancesForm } from '../../lib/messages'
+import { useIntl } from 'react-intl'
+
+import { RadioController } from '@island.is/shared/form-fields'
+import { Controller, useFormContext } from 'react-hook-form'
+import { HomeCircumstances } from '@island.is/financial-aid/shared/lib'
+import * as styles from '../Shared.css'
+import cn from 'classnames'
+import { FAFieldBaseProps, InputTypes } from '../../lib/types'
+
+const HomeCircumstancesForm = ({
+  application,
+  field,
+  errors,
+}: FAFieldBaseProps) => {
+  const typeInput = {
+    id: 'homeCircumstances.type',
+    error: errors?.homeCircumstances?.type,
+  } as InputTypes
+
+  const customeInput = {
+    id: 'homeCircumstances.custom',
+    error: errors?.homeCircumstances?.custom,
+  } as InputTypes
+
+  const { formatMessage } = useIntl()
+
+  const { answers } = application
+
+  const [statefulAnswer, setStatefulAnswer] = useState<
+    HomeCircumstances | undefined
+  >(answers?.homeCircumstances?.type)
+
+  const [statefulCustomInput, setstatefulCustomInput] = useState<string>(
+    answers?.homeCircumstances?.custom || '',
+  )
```

Comment:
Are these needed?

Reply:
I need one state to show the input, i think 🤔